tools/osbuilder/node-builder/azure-linux/common.sh

```diff
@@ -12,12 +12,14 @@ OS_VERSION=$(sort -r /etc/*-release | gawk 'match($0, /^(VERSION_ID=(.*))$/, a)
 
 ([[ "${OS_VERSION}" == "2.0" ]] || [[ "${OS_VERSION}" == "3.0" ]]) || die "OS_VERSION: value '${OS_VERSION}' must equal 3.0 (default) or 2.0"
 
+SHIM_CONFIG_LINK_NAME="configuration.toml"
 if [ "${CONF_PODS}" == "yes" ]; then
 	INSTALL_PATH_PREFIX="/opt/confidential-containers"
 	UVM_TOOLS_PATH_OSB="${INSTALL_PATH_PREFIX}/uvm/tools/osbuilder"
 	UVM_TOOLS_PATH_SRC="${INSTALL_PATH_PREFIX}/uvm/src"
 	UVM_PATH_DEFAULT="${INSTALL_PATH_PREFIX}/share/kata-containers"
-	IMG_FILE_NAME="kata-containers.img"
+	IMG_FILE_NAME="kata-containers-cc.img"
```

Comment:
We should not rename this imo. Otherwise we will need more changes higher up in the stack. Can discuss.